proxyd/consensus_poller.go

```diff
@@ -301,27 +376,75 @@ func (cp *ConsensusPoller) fetchBlock(ctx context.Context, be *Backend, block st
 
 	jsonMap, ok := rpcRes.Result.(map[string]interface{})
 	if !ok {
-		return 0, "", fmt.Errorf("unexpected response type checking consensus on backend %s", be.Name)
+		return 0, "", fmt.Errorf("unexpected response to eth_getBlockByNumber on backend %s", be.Name)
 	}
 	blockNumber = hexutil.Uint64(hexutil.MustDecodeUint64(jsonMap["number"].(string)))
 	blockHash = jsonMap["hash"].(string)
 
 	return
 }
 
-func (cp *ConsensusPoller) getBackendState(be *Backend) (blockNumber hexutil.Uint64, blockHash string) {
+// isSyncing Convenient wrapper to check if the backend is syncing from the network
+func (cp *ConsensusPoller) getPeerCount(ctx context.Context, be *Backend) (count uint64, err error) {
+	var rpcRes RPCRes
+	err = be.ForwardRPC(ctx, &rpcRes, "67", "net_peerCount")
+	if err != nil {
+		return 0, err
+	}
+
+	jsonMap, ok := rpcRes.Result.(string)
```

Comment:
Ditto here - isn't `RPCRes` a `json.RawMessage`, so it'll include the quotes if cast to a string?

Reply:
Same